packages/daimo-api/shovel/config.json

```diff
@@ -2,28 +2,18 @@
   "pg_url": "$DATABASE_URL",
   "eth_sources": [
     {
-      "name": "base",
-      "chain_id": 8453,
-      "url": "$BASE_RPC_URL",
-      "batch_size": 10,
-      "concurrency": 4
-    },
-    {
-      "name": "base-goerli",
-      "chain_id": 84531,
-      "url": "$BASE_GOERLI_RPC_URL",
-      "batch_size": 10,
-      "concurrency": 4
+      "name": "$CHAIN_NAME",
```

Comment:
could deduplicate this environment variable with `DAIMO_CHAIN` ?

Reply:
shovel runs in a seperate service so no env vars are shared, will just leave as is